src/web_interface/templates/macros.html

```diff
@@ -0,0 +1,20 @@
+{% macro start_stats_panel(panel_title, icon="chart-bar") -%}
+    <!-- stats: {{ panel_title }} -->
+    <div class="col-sm-6 col-md-4 mb-3 px-2">
+        <div class="card p-1 bg-light rounded border-0">
+            <div class="card-body bg-light">
+                <h5 class="card-title mb-3"><i class="fas fa-{{ icon }}"></i> {{ panel_title }}</h5>
+{%- endmacro %}
+
+{% macro end_stats_panel(counter) %}
```

Comment:
this seems not very elegant but there is a better way: [jinja calls](https://jinja.palletsprojects.com/en/2.11.x/templates/#call)
```html
{% macro stats_panel(panel_title, icon="chart-bar") -%}
    <!-- stats: {{ panel_title }} -->
    <div class="col-sm-6 col-md-4 mb-3 px-2">
        <div class="card p-1 bg-light rounded border-0">
            <div class="card-body bg-light">
                <h5 class="card-title mb-3"><i class="fas fa-{{ icon }}"></i> {{ panel_title }}</h5>
                {{ caller() }}
            </div>
        </div>
    </div>
{% endmacro %}
```

and in `show_statistics.html`:
```html
{% call macros.stats_panel("General") %}
    <table (...)/>
{% endcall %}
```

Reply:
I have to admit, I knew about this feature but I didn't really understand it